.github/workflows/integration_test.yml

```diff
@@ -219,9 +224,9 @@ jobs:
             sleep 1
           done
       - name: Run Java Integration Tests
-        if: inputs.test-goal != '' 
+        if: inputs.test-goal-java != '' 
         run: |
-          ./mvnw --errors --batch-mode --show-version -P${{ matrix.integration-test-profile }} ${{ inputs.maven_args_tests }} ${{ inputs.test-goal }}
+          ./mvnw --errors --batch-mode --show-version -P${{ matrix.integration-test-profile }} ${{ inputs.maven_args_tests }} ${{ inputs.test-goal-java }}
       - name: Checkout killbill-integration-tests
```

Comment:
You can also skip this if `inputs.test-goal-ruby != '' `

Reply:
Fixed